executor/api/rabbitmq/server.go

```diff
@@ -201,13 +204,39 @@ func (rs *SeldonRabbitMQServer) predictAndPublishResponse(
 		rs.Log.Error(err, UNHANDLED_ERROR)
 		return fmt.Errorf("unhandled error %w from predictor process", err)
 	}
+	arrBytes, e := resPayload.GetBytes()
+	if e == nil {
+		annotations, _ := k8s.GetAnnotations()
+		msgLimit := annotations["seldon.io/rabbitmq-max-message-size-in-bytes"]
```

Comment:
can you create the annotation as a constant here https://github.com/dominodatalab/seldon-core/blob/master/executor/k8s/annotation.go#L14?